packages/help-center/src/happychat-window-communicator.ts

```diff
@@ -4,69 +4,71 @@
  */
 import { useHappychatAuth } from '@automattic/happychat-connection';
 import { useDispatch, useSelect } from '@wordpress/data';
-import { useEffect } from 'react';
+import { useEffect, useState } from '@wordpress/element';
 import { useQueryClient } from 'react-query';
 import { useSelector } from 'react-redux';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 /**
  * Internal Dependencies
  */
 import { HELP_CENTER_STORE, SITE_STORE } from './stores';
-import type { WindowState } from './types';
 
-/**
- * This hook attaches an event listener to the window to listen to Happychat messages that com from https://widgets.wp.com/calypso-happychat/
- * It collects information from the help-center store and forwards it to Happychat. It also forwards the unread count and the state changes (ended, blurred, closed, opened) to whoever uses the hook
- *
- * @param onStateChange a callback that will be called whenever state changes
- * @param onUnreadChange a callback that will be called with the number of new unread messages
- */
```

Comment:
Why delete this?

Reply:
This one must stay, maybe improved, because this hook is not a normal hook and needs some explanation.